packages/features/eventtypes/components/tabs/availability/EventAvailabilityTab.tsx

```diff
@@ -76,48 +77,14 @@ const EventTypeScheduleDetails = memo(
       },
       { enabled: !!scheduleId || !!loggedInUser?.defaultScheduleId }
     );
-
-    const weekStart = weekStartNum(loggedInUser?.weekStart);
-
-    const filterDays = (dayNum: number) =>
-      schedule?.schedule.filter((item) => item.days.includes((dayNum + weekStart) % 7)) || [];
-
     return (
       <div>
         <div className="border-subtle space-y-4 border-x p-6">
-          <ol className="table border-collapse text-sm">
-            {weekdayNames(i18n.language, weekStart, "long").map((day, index) => {
-              const isAvailable = !!filterDays(index).length;
-              return (
-                <li key={day} className="my-6 flex border-transparent last:mb-2">
-                  <span
-                    className={classNames(
-                      "w-20 font-medium sm:w-32 ",
-                      !isAvailable ? "text-subtle line-through" : "text-default"
-                    )}>
-                    {day}
-                  </span>
-                  {isPending ? (
-                    <SkeletonText className="block h-5 w-60" />
-                  ) : isAvailable ? (
-                    <div className="space-y-3 text-right">
-                      {filterDays(index).map((dayRange, i) => (
-                        <div key={i} className="text-default flex items-center leading-4">
-                          <span className="w-16 sm:w-28 sm:text-left">
-                            {format(dayRange.startTime, timeFormat === 12)}
-                          </span>
-                          <span className="ms-4">-</span>
-                          <div className="ml-6 sm:w-28">{format(dayRange.endTime, timeFormat === 12)}</div>
-                        </div>
-                      ))}
-                    </div>
-                  ) : (
-                    <span className="text-subtle ml-6 sm:ml-0">{t("unavailable")}</span>
-                  )}
-                </li>
-              );
-            })}
-          </ol>
+          {schedule && schedule.timeBlocks?.length ? (
```

Comment:
Can we show both? Same we show now with the available days and below the Time blocks?

Reply:
Made the changes.